leases/services/invoice.py

```diff
@@ -352,3 +357,33 @@ def get_failed_orders(season_start: date) -> QuerySet:
         return Order.objects.filter(
             _lease_object_id__in=leases, status=OrderStatus.ERROR
         )
+
+
+class WinterStorageInvoicingService(BaseInvoicingService):
```

Comment:
I think it's great that the WinterStorageInvoicingService is so small, and so much common code is in BaseInvoicingService :-)